scripts/tasks.sh

```diff
@@ -855,9 +855,9 @@ function task_requires_root() {
     fi
 }
 
-# check if containerd on the current node has the `docker.io/rancher/mirrored-flannelcni-flannel:v<version>` image
+# check if containerd on the current node has the `docker.io/flannel/flannel` image
 function flannel_images_present() {
-    if ! ctr -n=k8s.io images ls | grep -q "docker.io/rancher/mirrored-flannelcni-flannel" ; then
+    if ! ctr -n=k8s.io images ls | grep -q "docker.io/flannel/flannel" ; then
```

Comment:
can we check for either of the two images instead?

Reply:
that way we don't break this migration for previous flannel versions

Reply:
Good point. If you're migrating to flannel versions `0.20.0`, `0.20.1`,  and `0.20.2` then those use the previous image name.